MOBILE_CLIENT/ANDROID/_LOW/APK_HAS_FRAGILE_USER_DATA_NOT_SET/recommendation.md

```diff
@@ -1,4 +1,6 @@
-Explicitly set the attribute `android:hasFragileUserData` to the appropriate value.
+The hasFragileUserData flag can be added to the application node of AndroidManifest.xml. 
+####
+If its value is true, then when the user uninstalls the app, a prompt will be shown to the user asking him whether to keep the app's data.
```

Comment:
This explanation should be in the description, not in the recommendation part. Because now in the description I do not understand what is the purpose of this flag